### PR TITLE
add type for batchnorm

### PR DESCRIPTION
**Pull request by yanndupis**
Hey @bendecoste,

This PR should add type for batchnorm.

Thanks

### CODE REVIEW

File: tensorflow_encrypted/layers/batchnorm.py [hunk 1]
```diff
@@ -4,15 +4,16 @@
 
 
 class Batchnorm(core.Layer):
-    def __init__(self, mean, variance, scale, offset, variance_epsilone=1e-8):
+    def __init__(self, mean: np.array, variance: np.array, scale: np.array,
+                 offset: np.array, variance_epsilone: float = 1e-8) -> None:
```
Comment (jvmancuso) [inline tensorflow_encrypted/layers/batchnorm.py]: Is `variance_epsilone` a typo?

Comment (yanndupis) [reply tensorflow_encrypted/layers/batchnorm.py]: oups yes, good catch! It should be `variance_epsilon`  https://www.tensorflow.org/api_docs/python/tf/nn/batch_normalization

**Comment by yanndupis** [comment]
Just fixed the typo. thanks,

**Comment by yanndupis** [comment]
The type should be fixed now. thanks
